library/src/amd_detail/rocblaslt/src/rocblaslt_mat.cpp

```diff
@@ -936,6 +943,11 @@ rocblaslt_status rocblaslt_gemm_create_cpp_impl_2(const rocblaslt_handle
         setTo1(compute_type, (void*)alpha_1, (const void**)&alpha);
     }
 
+    if constexpr(std::is_same<Inputs, rocblaslt::RocGemmInputsV2>::value)
+    {
+        amaxD = inputs.amaxD;
+    }
+
```

Comment:
Though we are not using RocGemmInputs anymore, but this is a template function so it'll be compiled still. So need to use constexpr to check the type otherwise we'll have compilation error. (amaxD is inited as nullptr above)